praxis-isncsci/ui
# Scaffold web app

Closes #50 

## Problem

We need a component that will manage the business logic and user interaction and will call the appropriate use cases to update the state of the application.

## Solution

The web app component adds the needed event listeners and controllers to manage user interaction. It exposes the initialize method where different implementations can pass in their own providers.

## Testing

- [x] Can view web app

<details>
  <summary>Test case</summary>

  1. Navigate to **Storybook** story `?path=/docs/app-app--docs` 
  2. Confirm that the web app is visible
  3. Click on the `calculate` button
  4. Confirm that the totals dialog is shown

</details>
